systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaResource.java

```diff
@@ -74,6 +77,12 @@ public void delete(Kafka resource) {
 
         kafkaClient().inNamespace(namespaceName).withName(
             resource.getMetadata().getName()).withPropagationPolicy(DeletionPropagation.FOREGROUND).delete();
+
+        // additional deletion of pvcs with specification deleteClaim set to false which were not deleted prior this method
+        for (PersistentVolumeClaim pvc : kubeClient().listPersistentVolumeClaims(namespaceName, clusterName)) {
+            kubeClient().deletePersistentVolumeClaim(namespaceName, pvc.getMetadata().getName());
+            PersistentVolumeClaimUtils.waitForPersistentVolumeClaimDeletion(namespaceName, pvc.getMetadata().getName());
+        }
```

Comment:
Why not do it in two cycles?
1) Request deletion of all PVCs in one loop
2) Wait for deletion in the second loop

That way, if PVC deletion takes 10 seconds and you delete 10 PVCs, you will nto wait 100 seconds but only ~10 seconds.

Reply:
> I left on nit. But I wonder why this is needed. AFAIU the tests now run in a namespace per test-suite. And the namespace should be delete after the test suite is complete. That should delete the PVCs as well. So why is this needed in addition to that?

Because, there are edge cases, where for instance you install Strimzi by OLM. When you install Strimzi by OLM and you want to have a cluster-wide setting, it has to be installed in the specific namespace where global `OperatorGroup` is installed. Such namespace we do not delete because it would also delete global `OperatorGroup` and thus we wouldn't be able to deploy Operator.

Reply:
re two cycles: I haven't bumped into a situation where deletion takes this much time. If there is a problem with deletion it's always caught in waitingForDeletion. Deletion is almost instantly completed in every situation. So in my opinion it should not be necessary.

Reply:
If it is instant, you do not need to wait for it ... if you need to wait for it, you should split it to save time ;-)

Reply:
You are right, I'm adding proposed solution for time save. Thank you for the suggestion.